default/sql-exporter.yml

```diff
@@ -4,7 +4,7 @@ global:
   # timing out first.
   scrape_timeout_offset: 500ms
   # Minimum interval between collector runs: by default (0s) collectors are executed on every scrape.
-  min_interval: 0s
+  min_interval: 300s
```

Comment:
We should probably align this with prometheus's scrape_interval too (and grafana's $__interval when fetching the data points).